components/Forms/QuestionLabel.tsx

```diff
@@ -33,6 +32,7 @@ export const QuestionLabel: React.FC<QuestionLabelProps> = ({
       </label>
       {helpText && (
         <span
+          id={`help-text-${fieldId}`}
```

Comment:
un-intented consequences? 
![image](https://user-images.githubusercontent.com/52539578/225078506-a9cb7313-ff1b-4a7b-b2fa-b4d6703bb12a.png)

Reply:
Fixed by checking the fieldId value